pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -274,9 +274,10 @@ func generateNetworkInterfaces(options *amifamily.LaunchTemplate, clusterIPFamil
 				//nolint: gosec
 				NetworkCardIndex: lo.ToPtr(int32(i)),
 				// Some networking magic to ensure that one network card has higher priority than all the others (important if an instance needs a public IP w/o adding an EIP to every network card)
-				DeviceIndex:   lo.ToPtr(lo.Ternary[int32](i == 0, 0, 1)),
-				InterfaceType: lo.ToPtr(string(ec2types.NetworkInterfaceTypeEfa)),
-				Groups:        lo.Map(options.SecurityGroups, func(s v1.SecurityGroup, _ int) string { return s.ID }),
+				DeviceIndex:     lo.ToPtr(lo.Ternary[int32](i == 0, 0, 1)),
+				InterfaceType:   lo.ToPtr(string(ec2types.NetworkInterfaceTypeEfa)),
+				Ipv4PrefixCount: lo.Ternary(clusterIPFamily == corev1.IPv6Protocol, nil, aws.Int32(1)),
```

Comment:
I don't think we can outright do this -- not everyone will want 16 IPs taken from their subnet out-of-the-box. For customers with small subnets or lots of instances with only a single pod on them, this is going to take away a lot of IPs and potentially lead to IP exhaustion